tox.ini

```diff
@@ -29,6 +29,8 @@ deps =
     oldestdeps: minimum_dependencies
 
     parallel: pytest-xdist
+
+    gwcs: gwcs>=0.22.0
```

Comment:
The oldest version of gwcs with the `examples` module needs asdf 3.3 so we can't have it in the mix with the rest of the oldestdeps build. Doing this makes it optional and hopefully makes it easier to test old asdfs etc.

Reply:
Thanks. It sounds like there are other reasons to bump to 3.3 (2.15 is ancient and https://github.com/astropy/asdf-astropy/pull/296#discussion_r2336658866 will allow for just the public class path).

I'll open a PR to bump that pin to see if it triggers any other changes.